client/components/drop-zone/index.jsx

```diff
@@ -25,11 +25,12 @@ import {
 
 export const DropZone = React.createClass( {
 	propTypes: {
+		customClass: PropTypes.string,
```

Comment:
Just for more consistency, I would be all for using the general `className` here instead of `customClass` (not to be found anywhere else in Calypso). We do it in other places where we want to pass a custom class that the component will add to their internal styling (`Card` component just to name one).